tests/test_freshdesk_pagination.py

```diff
@@ -1,58 +1,95 @@
-from tap_tester import menagerie, connections, runner
-import re
+from math import ceil
+from tap_tester import connections, runner, LOGGER
 
 from base import FreshdeskBaseTest
 
+
 class PaginationTest(FreshdeskBaseTest):
 
     def name(self):
-        return "tap_freshdesk_pagination_test"
+        return "tap_tester_freshdesk_pagination_test"
 
     def test_name(self):
-        print("Pagination Test for tap-freshdesk")
-
+        LOGGER.info("Pagination Test for tap-freshdesk")
+    
+    def get_properties(self, *args, **kwargs):
+        """Override properties by passing page_size param."""
+        props = super().get_properties(*args, **kwargs)
+        props['page_size'] = self.PAGE_SIZE
+        return props
+    
     def test_run(self):
+        """
+        • Verify that for each stream you can get multiple pages of data.  
+        This requires we ensure more than 1 page of data exists at all times for any given stream.
+        • Verify by pks that the data replicated matches the data we expect.
+        """
 
-        # instantiate connection
-        conn_id = connections.ensure_connection(self)
+        # For roles stream data present in test account is limited. So, adding configurable page_size "2"
+        streams_to_test_1 = {"roles"}
+        self.run_test(streams_to_test_1, 2)
 
-        # Add supported streams 1 by 1
-        streams_to_test = {'agents', 'tickets'}
+        streams_to_test_2 = self.expected_streams(only_trial_account_streams=True) - streams_to_test_1
+        self.run_test(streams_to_test_2, 100)
 
-        # Run check mode
-        # Check mode has no catalog discovery for freshdesk
-        check_job_name = self.run_and_verify_check_mode(conn_id)
+    def run_test(self, streams_to_test, page_size):
```

Comment:
Rename the function argument from `streams_to_test` to `expected_streams`. And remove `expected_streams = streams_to_test` from below.

Reply:
Updated code as per the above suggestion.